### PR TITLE
Update libvirt and related packages

### PR DESCRIPTION
**Pull request by Hoshpak**
As requested in #5398, this pull request updates libvirt and related packages (libvirt-glib, libvirt-python) to their respective current version. I also took the liberty to add a logger to libvirtd since it has no logging configured in the default configuration and thus spams tty1.

### CODE REVIEW

**Comment by asergi** [comment]
Please rebase.

**Comment by Hoshpak** [comment]
rebased on top of master and dropped the libvirt-python commit since the package has been updated in the meantime.

File: srcpkgs/libvirt-glib/template [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
+version=1.0.0
 revision=1
 build_options="gir"
 build_style=gnu-configure
 configure_args="--disable-static $(vopt_enable gir introspection)"
 hostmakedepends="automake libtool pkg-config intltool glib-devel
  gettext-devel gtk-doc gobject-introspection vala-devel python-devel"
 makedepends="libglib-devel libvirt-devel libxml2-devel python-devel"
-short_desc="libvirt glib integration"
+short_desc="Libvirt glib integration"
```
Comment (Gottox) [inline srcpkgs/libvirt-glib/template]: ignore the linter in this situation, lowercase is ok.

File: srcpkgs/libvirt-glib/template [hunk 2]
```diff
@@ -38,11 +37,3 @@ libvirt-glib-devel_package() {
...
 	}
 }
-libvirt-glib-python_package() {
-	lib32disabled=yes
-	depends="python"
-	short_desc+=" - python bindings"
-	pkg_install() {
-		vmove "usr/lib/python*"
-	}
-}
```
Comment (Gottox) [inline srcpkgs/libvirt-glib/template]: to make sure, that older versions of libvirt-glib-python won't interfer with this update, make sure you set `replaces="libvirt-glib-python>=0"`

File: srcpkgs/libvirt-glib/template (hunk 1 above)
Comment (Hoshpak) [inline srcpkgs/libvirt-glib/template]: OK, reverted that.

File: srcpkgs/libvirt-glib/template (hunk 2 above)
Comment (Hoshpak) [inline srcpkgs/libvirt-glib/template]: Rebased on top of master and added the replaces definition.

**Comment by bra1nwave** [comment]
Please update `libvirt` to 3.1.0. Thank you.

**Comment by Hoshpak** [comment]
I needed to rebase the branch anyway because of the virt-manager update, now it updates libvirt to 3.1.0. I'd appreciate it if someone would merge this PR. I don't usually mind rebasing PRs but the time I spent doing this every few days could be spent more effectively.

**Pull request closed without merging**